docs/source/conf.py

```diff
@@ -4,7 +4,7 @@
 
 import pyvista
 import numpy as np
-from pyansys_sphinx_theme import pyansys_logo_black
```

Comment:
What's the differences between pyansys_sphinx_theme and ansys_sphinx_theme  ?

Reply:
The pyansys_sphinx_theme is the older, deprecated project, and ansys_sphinx_theme is the new one the pyansys libraries are supposed to use. https://github.com/ansys/ansys-sphinx-theme